pricepoint_intel/ingestion/api_connector.py

```
@@ -0,0 +1,774 @@
+                        )
+                        session.add(history)
+
+                stats.records_saved += 1
+
+        stats.end_time = datetime.utcnow()
+        return stats
+
+    async def fetch_bulk_pricing(
+        self,
+        endpoint: str = "/pricing/bulk",
+        sku_ids: list[str] = None,
```

Ruff (0.14.6)

PEP 484 prohibits implicit `Optional`

Convert to `Optional[T]`

(RUF013)